zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/SessionsThreadLocalImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.github.onsdigital.zebedee.session.store;
+
+import com.github.onsdigital.zebedee.session.store.exceptions.*;
+
+import com.github.onsdigital.exceptions.*;
+
+import com.github.onsdigital.impl.*;
+import com.github.onsdigital.interfaces.*;
+import javax.servlet.http.HttpServletRequest;
+import com.github.onsdigital.JWTHandlerImpl;
+
+/**
+ * This method creates a ThreadLocal for the validated jwt 
+ * from a HTTP request and appropriate key 
+ * as part of implementing dp-identity-api
+ */
+
+public class SessionsThreadLocalImpl implements SessionsThreadLocal {
+
+    private static HttpServletRequest request;
+    private static String secretKey;
+    private static ThreadLocal<UserDataPayload> store =
+            new ThreadLocal<>();
+
+    private JWTHandler jwtHandler = new JWTHandlerImpl();
```

Comment:
We want to avoid setting values like this within the class because it limits how we can setup this object.

Add a new constructor:
```
public SessionsThreadLocalImpl(JWTHandler jwtHandler) {
    this.jwtHandler = jwtHandler
}
```

By doing so its the responsibility of the caller to provide this dependency. This is important as it allows us flexibility - in our tests we can provide a mocked JWTHandler. This allows us to test this object without having to worry about providing real tokens etc. We can configure the mock to response however we require.